lua/tests/example_project/app/src/test/kotlin/org/example/KotestDescribeExample.kt

```diff
@@ -1,5 +1,5 @@
 
-package kotlin.org.example
+package org.example
```

Comment:
Using the `kotlin` package prefix causes Gradle to fail the build immediately saying that only the Kotlin team/compiler can use the kotlin prefix.